torchtext/prototype/t5/modules.py

```diff
@@ -74,3 +77,60 @@ def _compute_bias(
     values = relative_attention_bias(relative_position_bucket)  # shape (query_length, key_length, num_heads)
     values = values.permute([2, 0, 1]).unsqueeze(0)  # shape (1, num_heads, query_length, key_length)
     return values
+
+
```

Comment:
`_t5_scaled_dot_product_attention()` is modified from [pytorch implementation](https://github.com/pytorch/pytorch/blob/5953fd9133c0bdcc0158acf1472fac403bc5f636/torch/nn/functional.py#L4814) to incorporate relative attention bias when computing the attention scores. `position_bias` is passed in as a parameter (`line 87`) and is added to the attention score computation in lines `116-121, 126, 130`